pkgs/applications/version-management/sourcehut/default.nix

```diff
@@ -0,0 +1,49 @@
+{ python37, openssl_1_1
+, callPackage }:
+
+# To expose the *srht modules, they have to be a python module so we use `buildPythonModule`
+# Then we expose them through all-packages.nix as an application through `toPythonApplication`
+# https://github.com/NixOS/nixpkgs/pull/54425#discussion_r250688781
+
+let
+  fetchNodeModules = callPackage ../../networking/instant-messengers/rambox/fetchNodeModules.nix { };
+
+  python = python37.override {
+    packageOverrides = self: super: {
+      srht = self.callPackage ./core.nix { inherit fetchNodeModules; };
```

Comment:
Is there a way to expose this package within a nix-shell? (Context: trying to do development for builds.sr.ht on NixOS.)

Reply:
@eadwu since you are listed as the maintainer

Reply:
`sourcehut.python.pkgs.srht`

Reply:
In case you switch threads, please mention me. I'm trying to keep an eye on the progress of this.

Reply:
Oh, wow, I feel really dumb now. I missed that `python` was inherited. Thanks for the quick response!